src/core/server/saved_objects/migrations/opensearch_dashboards/opensearch_dashboards_migrator.test.ts

```diff
@@ -76,6 +77,12 @@ describe('OpenSearchDashboardsMigrator', () => {
       const mappings = new OpenSearchDashboardsMigrator(options).getActiveMappings();
       expect(mappings).toMatchSnapshot();
     });
+
+    it('permissions field exists in the mappings when the feature is enabled', () => {
```

Comment:
do we cover the tests that permissons field not exists in the mapping when the feature is disabled? Thanks

Reply:
Yes, the mocked value of the feature flag is false for all other tests, and they will check if the generated mappings match the snapshot which doesn't contain the `permissions` field: ` expect(mappings).toMatchSnapshot();`.